src/main/java/sirius/db/es/ElasticQuery.java

```diff
@@ -463,6 +465,17 @@ public ElasticQuery(EntityDescriptor descriptor, LowLevelClient client) {
         return this;
     }
 
+    @Override
+    public Stream<E> stream() {
+        // just in case no absolute ordering is present
+        orderAsc(Mapping.named(KEY_DOC_ID));
```

Comment:
check if orderings are present? this might kill an index if it is appended to an existing ordering (?)

Reply:
Why would it kill an index? if you have `ORDER BY a, b, id`, and you have an index `a, b, c` this should be used by the database, as far as I know, and the "remaining" ordering by ID is done during evaluation.
However, if I leave the ID out, this might lead to inconsistencies if `ORDER BY a, b` is not a total ordering.
I do not know about the elastic implementation, though.